src/style.css

```diff
@@ -1,3 +1,74 @@
+html {
+  font-family: Roboto, serif;
+}
+
 body {
   margin: 0;
+  --background-color: #fff;
+  --primary_color: #00acdc;
+  --text-color: #000;
+}
+
+.header__moyo {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background-color: var(--background-color);
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.company__logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav__block {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
```

Comment:
1. `flex-direction: row` is the default value, no need to specify it.
2. I recommend moving reseting styles to the top and apply them for all `<ul>` (Usually you don't want those padding and bullets in every list)
```
ul {
  list-style: none;
   padding: 0;
   margin: 0;
}
```